arkane/sensitivity.py

```diff
@@ -454,7 +454,10 @@ def plot(self, wells, transition_states):
                 axis.set_xlabel(r'Sensitivity: $\frac{\partial\:\ln{k}}{\partial\:E0}$, ($\frac{J}{mol}$)')
                 # axis.ticklabel_format('sci')
                 axis.set_title('{0}, {1}'.format(condition[0], condition[1]))
-                axis.set_xlim([min_sa, max_sa])
+                try:
+                    axis.set_xlim([min_sa, max_sa])
+                except:
+                    pass
```

Comment:
Should we be throwing warning with NaN sensitivity values or are we throwing it somewhere else already?

Reply:
Plotting happens last so I wouldn't worry too much about NaN handling. If plotting fails they'll be able to see the NaN's in the sensitivity output.